scripts/release_notes_generator.py

```diff
@@ -23,6 +23,14 @@
     'feature request',
     'customer success',
 ]
+ISSUE_LABELS_ORDERED_BY_IMPORTANCE = [
```

Comment:
Would it be better to just change the order of `ISSUE_LABELS` to match what you want, rather than creating a new global variable?

Reply:
The other one is a different order because when picking which category to assign a label to, I want the order to be different. For example, if an issue is labeled both a feature request and internal, I want it to be only put in the internal category of the release notes. So the order for `ISSUE_LABELS` is intentionally different